src/server/src/main/java/io/cassandrareaper/core/RepairSchedule.java

```diff
@@ -37,13 +37,13 @@
   private final int daysBetween;
   private final DateTime nextActivation;
   private final ImmutableList<UUID> runHistory;
-  @Deprecated private final int segmentCount;
```

Comment:
Why is `segmentCount` getting removed?

Reply:
It's been deprecated for a while and as I was working on segment count per node with this PR, I figured it was the right time to do some cleanup. This `segmentCount` variable is the former "global segment count" which we replaced with the segmentCountPerNode to have schedules self adjust to cluster expansions.